packages/server/postgres/migrations/1709551949071_makeAllTemplatesFree.ts

```diff
@@ -0,0 +1,65 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(
+    `
+    UPDATE "MeetingTemplate"
+      SET "isFree" = true
+   `
+  )
+
+  await client.end()
+}
+
+const PREV_FREE_TEMPLATES = [
```

Comment:
I think that's because the first query in the down migration didn't check where orgId is `aGhostOrg`, so all of the custom templates were set to non-free. 

I've updated the down migration and checked staging for the list of free template ids.